index.js

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios'
+
+import {
+    getInput,
+    setFailed,
+    getMultilineInput,
+    setOutput,
+} from '@actions/core'
+import { context } from '@actions/github'
+
+const appId = getInput('app-id')
+const workflowId = getInput('workflow-id')
+const token = getInput('token')
+
+const labels = getMultilineInput('labels')
+
+let branch = getInput('branch')
+let tag = getInput('tag')
+
+if (!branch && !tag) {
+    const { ref } = context
+    branch = ref.split('refs/heads/')[1] || process.env.GITHUB_HEAD_REF
+    tag = ref.split('refs/tags/')[1]
+}
+
+const variables = {}
+Object.entries(process.env).forEach(([key, value]) => {
+    if (key.startsWith('CM_')) {
```

Comment:
I think this suffix can create confusing later on Codemagic side. 
One option to remove this suffix when we send it to Codemagic. And users can see difference between built-in and custom variables.
or find some better suffix.